.github/workflows/workspace_integration_test.yaml

```diff
@@ -1,12 +1,79 @@
-name: Reusable workflow for building Docker images
+name: Workspace Integration Test
 
 on:
   workflow_call:
+    inputs:
+      image_tag:
+        type: string
+    secrets:
+      studio_license_key:
+        required: false
 
 jobs:
-  job_name:
-    name: Any job name with
+  setup:
     runs-on: ubuntu-latest
+    outputs:
+      #We return the image_tag input of the workflow_call or the dotenv action if it is not provided
+      image_tag: ${{ github.event.inputs.image_tag || steps.dotenv.outputs.studio_docker_tag }}
     steps:
-      - name: Hello World
-        run: echo "Hello World after"
\ No newline at end of file
+      - uses: actions/checkout@v4
+      - id: dotenv
+        uses: falti/dotenv-action@v1.0.4
+  integration-test-in-studio-container:
+    needs: setup
+    runs-on: ubuntu-latest
+    env:
+      STUDIO_LICENSE_KEY: ${{ secrets.studio_license_key || secrets.STUDIO_CI_LICENSE_KEY }}
+      # TODO this variable is required but github workspace is different
+      # see https://github.com/actions/runner/issues/2058
+      USER_WS: /github/workspace
+      ROS_DOMAIN_ID: 49
+      ROS_LOCALHOST_ONLY: 1
+      RMW_IMPLEMENTATION: rmw_cyclonedds_cpp
+      CYCLONEDDS_URI: /tmp/cyclonedds.xml
+    # TODO: docker tag
+    # TODO: example for custom base images
+    container: picknikciuser/moveit-studio:${{ needs.setup.outputs.image_tag }}
```

Comment:
Isn't the docker tag already specified in this line? Is the TODO outdated?

Reply:
yep. thx
d05209c0ad2c67d16e2bed6a9d7493f6628f3430